align_anything/configs/template.py

```diff
@@ -1876,27 +1668,161 @@ def format_prompt_only_sample(self, raw_sample: dict[str, Any]) -> dict[str, Any
 
         return {'conversation': conversation}
     
-@register_template('Qwen2Audio')
-class Qwen2Audio:
```

Comment:
If this Template is deleted, should the content in the [Model-Zoo](https://github.com/PKU-Alignment/align-anything/blob/main/Model-Zoo.md) be modified, or should the [Model-Zoo](https://github.com/PKU-Alignment/align-anything/blob/main/Model-Zoo.md) be deleted?